plugins/modules/zos_copy.py

```diff
@@ -918,7 +920,7 @@ def copy_to_seq(
             if self.force_lock:
                 copy_args["options"] += " -f"
 
-            response = datasets._copy(new_src, dest, None, **copy_args)
+            response = datasets._copy(new_src, dest, **copy_args)
```

Comment:
hey could you review if is possible to remove underscore zoau functions ? Not saying is wrong or bad but do remember Demetri said to remove if possible.

Reply:
Removed all calls to hidden ZOAU functions and ran the pipeline again for zos_copy.

![Screenshot 2024-02-16 at 16 19 14](https://github.com/ansible-collections/ibm_zos_core/assets/17365764/8ea51124-9d87-435e-94cb-52d0e5d8aae6)
![Screenshot 2024-02-16 at 16 19 55](https://github.com/ansible-collections/ibm_zos_core/assets/17365764/7645a562-70e3-477e-801d-89276c803c06)

Reply:
And for zos_job_submit

![Screenshot 2024-02-16 at 16 33 38](https://github.com/ansible-collections/ibm_zos_core/assets/17365764/508c4099-f6e9-4ea9-b2b2-717b4f7b60ed)
![Screenshot 2024-02-16 at 16 33 53](https://github.com/ansible-collections/ibm_zos_core/assets/17365764/3584a97c-e5a5-41a4-be81-7db78c89de6a)